vendor/bat-native-ledger/include/bat/ledger/ledger_callback_handler.h

```diff
@@ -35,9 +35,9 @@ LEDGER_EXPORT enum Result {
   TIP_ERROR = 16,
   CORRUPTED_WALLET = 17,
   GRANT_ALREADY_CLAIMED = 18,
-  // some more useful result codes should go here
 
   LEDGER_RESULT_END = 19,
```

Comment:
Confusing as no longer the last enum???

Reply:
yeah I removed that text as it was not at the end and it's not really useful